src/ir_Technibel.cpp

```diff
@@ -338,21 +315,16 @@ bool IRTechnibelAc::getSwing(void) {
 /// @return true, the swing is on. false, the swing is off.
 bool IRTechnibelAc::convertSwing(const stdAc::swingv_t swing) {
   switch (swing) {
-    case stdAc::swingv_t::kOff:
-      return false;
-    default:
-      return true;
+    case stdAc::swingv_t::kOff: return false;
+    default:                    return true;
```

Comment:
Could this be changed to `return swing != stdAc::swingv_t::kOff;` ?

Reply:
Why yes. Yes it can. Well spotted! Done.